packages/nbdime/src/merge/widget/metadata.ts

```diff
@@ -36,18 +36,27 @@ export class MetadataMergeWidget extends MergePanel<MetadataMergeModel> {
 
     // We know/assume that MetadataMergeModel never has
     // null values for local/remote:
+    const viewOptions = {...this._viewOptions};
+    const unchanged = model.decisions.length === 0;
+    if (unchanged) {
+      viewOptions.collapseIdentical = false;
+    }
     this.view = createNbdimeMergeView({
       remote: model.remote,
       local: model.local,
       merged: model.merged,
       factory: this._editorFactory,
       translator: this._translator,
-      ...this._viewOptions,
+      ...viewOptions,
     });
     const trans = this._translator.load('nbdime');
     const wrapper = new CollapsiblePanel(
       this.view,
-      trans.__('Notebook metadata changed'),
+      trans.__(
+        `Notebook metadata ${
+          unchanged ? 'unchanged' : 'changed'
+        }`,
+      ),
```

Comment:
This needs to be two separate strings or use %1 with separate `trans.__()` wrappers for each unchanged/changed words. It is slightly easier for translators if they have two full sentences rather than three strings they need to understand how they match together.

```suggestion
      unchanged ? trans.__('Notebook metadata unchanged') : trans.__('Notebook metadata changed'),
```